src/black/__init__.py

```diff
@@ -1315,7 +1315,9 @@ def assert_equivalent(src: str, dst: str, *, pass_num: int = 1) -> None:
         src_ast = parse_ast(src)
     except Exception as exc:
         raise AssertionError(
-            f"cannot use --safe with this file; failed to parse source file: {exc}"
+            f"cannot use --safe with this file; failed to parse source file AST: {exc}\n"
+            f"This could be caused by running black with an older python version "
```

Comment:
```suggestion
            f"cannot use --safe with this file; failed to parse source file AST: "
            f"{exc}\n"
            f"This could be caused by running black with an older python version "
```
Line too long for lint